src/resources/js/v2/tickets-commerce.js

```diff
@@ -0,0 +1,166 @@
+ * Configures ET Tickets Commerce Object in the Global Tribe variable
+ *
+ * @since TBD
+ *
+ * @type   {PlainObject}
+ */
+tribe.tickets.commerce = {};
+
+/**
+ * Initializes in a Strict env the code that manages the plugin tickets commerce.
+ *
+ * @since TBD
+ *
+ * @param  {PlainObject} $   jQuery
+ * @param  {PlainObject} obj tribe.tickets.commerce
+ *
+ * @return {void}
+ */
+( function( $, obj ) {
+	'use strict';
+	const $document = $( document );
+
+	/*
+	 * Tickets Commerce Selectors.
+	 *
+	 * @since TBD
+	 */
+	obj.selectors = {
+		checkoutContainer: '.tribe-tickets__commerce-checkout',
+		checkoutItem: '.tribe-tickets__commerce-checkout-cart-item',
+		checkoutItemDescription: '.tribe-tickets__commerce-checkout-cart-item-details-description',
+		checkoutItemDescriptionOpen: '.tribe-tickets__commerce-checkout-cart-item-details--open',
+		checkoutItemDescriptionButtonMore: '.tribe-tickets__commerce-checkout-cart-item-details-button--more',
+		checkoutItemDescriptionButtonLess: '.tribe-tickets__commerce-checkout-cart-item-details-button--less',
+		hiddenElement: '.tribe-common-a11y-hidden',
+	};
+
+	/**
+	 * Toggle the checkout item description visibility.
+	 *
+	 * @since TBD
+	 *
+	 * @param {event} event The event.
+	 *
+	 * @return {void}
+	 */
+	obj.checkoutItemDescriptionToggle = function( event ) {
+		if ( 'keyup' === event.type && 13 !== event.keyCode ) {
+			return;
+		}
+
+		const trigger = event.target;
+
+		if ( ! trigger ) {
+			return;
+		}
+
+		const $trigger = $( trigger );
+
+		if (
+			! $trigger.hasClass( obj.selectors.checkoutItemDescriptionButtonMore.className() ) &&
+			! $trigger.hasClass( obj.selectors.checkoutItemDescriptionButtonLess.className() )
+		) {
+			return;
+		}
+
+		const $parent = $trigger.closest( obj.selectors.checkoutItem );
+		const $target = $( '#' + $trigger.attr( 'aria-controls' ) );
+
+		if ( ! $target.length || ! $parent.length ) {
+			return;
+		}
+
+		// Let our CSS handle the hide/show. Also allows us to make it responsive.
+		const onOff = ! $parent.hasClass( obj.selectors.checkoutItemDescriptionOpen.className() );
+		$parent.toggleClass( obj.selectors.checkoutItemDescriptionOpen.className(), onOff );
+		$target.toggleClass( obj.selectors.checkoutItemDescriptionOpen.className(), onOff );
+		$target.toggleClass( obj.selectors.hiddenElement.className() );
+	};
+
+	/**
+	 * Binds the checkout item description toggle.
+	 *
+	 * @since TBD
+	 *
+	 * @param {jQuery} $container jQuery object of the tickets container.
+	 *
+	 * @return {void}
+	 */
+	obj.bindCheckoutItemDescriptionToggle = function( $container ) {
+		const $descriptionToggleButtons = $container.find( obj.selectors.checkoutItemDescriptionButtonMore + ', ' + obj.selectors.checkoutItemDescriptionButtonLess );
+
+		// Add keyboard support for enter key.
+		$descriptionToggleButtons.on(
+			'keyup',
+			obj.checkoutItemDescriptionToggle
+		);
+
+		$descriptionToggleButtons.on(
+			'click',
+			obj.checkoutItemDescriptionToggle
+		);
```

Comment:
I honestly didn't implement the description toggle. I shared [in the PR description](https://github.com/the-events-calendar/event-tickets/pull/2052#issue-713547186) that I'm bringing that over from the implementation there is for the modal, because we're running late with the TC project.